scss/tech.scss

```diff
@@ -0,0 +1,54 @@
+$tech-box-border-radius: 10px;
+$tech-box-height: 450px;
+$tech-box-width: 300px;
+$tech-logo-size: 100px;
+$tech-text-margin: 20px;
+
+.tech-box {
+  background-color: $green;
+  border-radius: $tech-box-border-radius;  
+  display: flex;
+  flex-direction: column;
+  height: $tech-box-height;
+  margin: 35px;
+  padding: 40px;
+  position: relative;
+  text-align: center;
+  width: $tech-box-width;
+
+  &::before {
+    content: '';
+    background-color: $white;
+    border: 1px solid $green;
+    border-radius: $tech-box-border-radius;    
+    bottom: 10px;
+    height: $tech-box-height;
+    position: absolute;
+    right: 10px;
+    width: $tech-box-width;
+    z-index: -1;
+  }
+
+  .tech-box-logo {
+    margin: 0 auto $tech-text-margin;
+    width: $tech-logo-size;
+  }
+
+  .tech-box-title {
+    color: $black;
+    font-size: $box-title-size;
+    margin: 0 auto $tech-text-margin;
+  }
+
+  .tech-box-launch {
+    color: $black;
+    font-size: $box-launch-text-size;
+    font-weight: bold;
+    margin: 0 auto ($tech-text-margin * 2);
+  }
+
+  .tech-box-text {
+    color: $black;
```

Comment:
Since you have many `color: $black` in text labels, you can make something like

`.tech-box-text,
.text-box-launch,
tech-box-title {
  color: $black;
}`